lib/collections/collectionFS.js

```diff
@@ -64,3 +64,113 @@ export const Media = new FS.Collection("Media", {
     }
   }
 });
+
+export const Audio = new FS.Collection("Audio", {
+  stores: [new FS.Store.GridFS("audio", {
+    transformWrite: function (fileObj, readStream, writeStream) {
+      readStream.pipe(writeStream);
+    }
+  })
+  ],
+  filter: {
+    allow: {
+      contentTypes: ["audio/*"],
+      extensions: ["wav", "wma", "aac", "mp3"]
+    }
+  }
+});
+
+export const Video = new FS.Collection("Video", {
+  stores: [
+    new FS.Store.GridFS("video", {
+      transformWrite: function (fileObj, readStream, writeStream) {
+        readStream.pipe(writeStream);
+      }
+    })
+  ],
+  filter: {
+    allow: {
+      contentTypes: ["video/*"],
+      extensions: ["mp4", "mov", "flv"]
+    }
+  }
+});
+
+
+export const Book = new FS.Collection("Book", {
+  stores: [new FS.Store.GridFS("book", {
+    transformWrite: function (fileObj, readStream, writeStream) {
+      readStream.pipe(writeStream);
+    }
+  })
+  ]
+});
+
+export const Software = new FS.Collection("Software", {
+  stores: [new FS.Store.GridFS("software", {
+    transformWrite: function (fileObj, readStream, writeStream) {
+      readStream.pipe(writeStream);
+    }
+  })
+  ]
+});
+
+Audio.allow({
```

Comment:
refactor this code 
1. Keep it DRY
2. ensure the function is actually doing something if at all it is needed.